content/8.x/admin/releases/8.0-beta/_index.en.adoc

```diff
@@ -101,4 +101,80 @@ Please link:https://suitecrm.com/download[visit the official website] to find th
 
 To report any security issues please follow our Security Process and send them directly to us via email security@suitecrm.com
 
+'''
+
+== 8.0-beta-2
```

Comment:
beta 2 should be above beta one and be at the top

Reply:
Thanks! Looked past that one

Reply:
Fixed now